src/theme/base-styles.ts

```diff
@@ -1,6 +1,6 @@
 import {TextStyle} from 'react-native';
 
-import {colors} from '../constants/colors';
+import {colors} from './';
```

Comment:
This looks a bit odd. Can this be done via a more expressive import that uses aliases and/or a named file/module?